### PR TITLE
Ameba: Fix Lint/UselessAssign

### PR DESCRIPTION
**Pull request by syeopite**

### CODE REVIEW

File: src/invidious/channels/channels.cr [hunk 1]
```diff
@@ -232,7 +232,7 @@ def fetch_channel(ucid, pull_all_videos : Bool)
       id:                 video_id,
       title:              title,
       published:          published,
-      updated:            Time.utc,
+      updated:            updated,
```
Comment (syeopite) [inline src/invidious/channels/channels.cr:235]: I'm not too sure if this should be using the current time `Time.utc` or the parsed updated time

Comment (SamantazFox) [reply src/invidious/channels/channels.cr:235]: I think it's supposed to report the time when the channel content was fetched by the background job. That's especially useful for channels that don't post often, when the last update is for a very old video.

Comment (syeopite) [reply src/invidious/channels/channels.cr:235]: In that case should there be a new attribute that reports when the video was updated on YouTube's side? Or should we just remove the logic that parses the updated time?

Comment (SamantazFox) [reply src/invidious/channels/channels.cr:235]: Okay, my bad. All that time I though we were talking about `InvidiousChannel.updated` while you want to change `ChannelVideo.updated`.

My previous message applies to the former, and your fix in indeed welcome in the latter case :)

**Review by SamantazFox — APPROVED** [review]

**Comment by SamantazFox** [comment]
Thank you!

**Pull request merged**